static/sidebar.js

```diff
@@ -0,0 +1,23 @@
+Polymer({
+  is: "ufo-sidebar",
+  ready: function() {
+    this.pages = [
+    {"href": "/", "text": "Home"},
+    {"href": "/user", "text": "Users"},
```

Comment:
I'm going to leave it for now since it looks like we'll be switching that very soon.